scraper/github/queryManager.py

```diff
@@ -101,6 +103,21 @@ def maxRetry(self, maxRetry):
         self.__maxRetry = numIn
         print("Auto-retry limit for requests set to %d." % (self.maxRetry))
 
+    @property
+    def retryDelay(self):
+        """int: Number of seconds to wait between automatic request retries.
+
+        Must be a whole integer greater than 0.
+        """
+        return self.__retryDelay
+
+    @retryDelay.setter
+    def retryDelay(self, retryDelay):
```

Comment:
Given the need to be an int, should there be any input validation here?

Reply:
The `int()` function will ensure it is the right type (or throw an error if it doesn't know what to do with it). The line after will ensure it is a positive number, forcibly setting it to `1` if it isn't. It's the same logic we've had in place for the `maxRetry` option.

Ideally, we should be able to support fractions of seconds here, but that won't be possible until I revisit the fancy countdown display I wrote. (I plan to look at that today, since it really does spam the logs.)

Reply:
Sounds good. Do you want me to cut the release now, or hold off?

Reply:
Maybe hold off until later this afternoon / EOD. I probably won't implement the `float` support yet, but just making a cleaner countdown that isn't printing lines constantly should be an easy fix.